rllib/policy/torch_policy.py

```diff
@@ -185,6 +185,84 @@ def compute_actions(self,
             return convert_to_non_torch_type((actions, state_out,
                                               extra_fetches))
 
+    @override(Policy)
+    @DeveloperAPI
+    def _compute_actions_from_trajectories(
+            self,
+            trajectories: List[Trajectory],
+            other_trajectories: Dict[AgentID, Trajectory],
+            explore: bool = None,
+            timestep: Optional[int] = None,
+            **kwargs):
+
+        explore = explore if explore is not None else self.config["explore"]
+        timestep = timestep if timestep is not None else self.global_timestep
+
+        with torch.no_grad():
+            # Create a view and pass that to Model as `input_dict`.
+            input_dict = self._lazy_tensor_dict(
+                get_trajectory_view(self.model, trajectories, is_training=False))
+            # TODO: (sven) support RNNs w/ fast sampling.
```

Comment:
Isn't the only difference from the other compute actions the way we generate the input dict? Does it make sense to reuse the rest of the code after that point?

Reply:
We can share the code, sure.

Reply:
Can we do this in this pr?